### ISSUE TITLE
list.dirs doesn't have argument pattern

### ISSUE THREAD

**Issue by stanstrup** [sjewo/shiny-loadbalancer#1]
`list.dirs("../", pattern=paste0("^",pwd(),"_[0-9]"))`

throws an error: 
`unused argument (pattern = paste0("^", pwd(), "_[0-9]"))`

**Comment by stanstrup** [comment]
Sorry. didn't see you had replaced the function with your own.

**Comment by sjewo** [comment]
Thanks for your message. I changed the function name and added the missing java script file.

**Comment by stanstrup** [comment]
Is the script different from the original? Because I changed a lot of things in the shiny app to make it work for me. And now it has the right value in the input field. But no re-direction happens.
Instead I get `Uncaught TypeError: Cannot read property 'length' of undefined` with reference to the script.

**Comment by sjewo** [comment]
Which branch did you pull? If you're on the master branch, there is maybe a problem in shiny-monitor.R. The grep in line 27 looks for "VERBUNDEN" in the netstat output. In an english environment you must change it to "CONNECTED" (check netstat for the right heading).

The script is fairly buggy and its a lot of CPU, so I switched to the fairly-scheduler branch. I added a short introduction to the README.md.
